cypress/fixtures/quests/list_of_user_quests.json

```diff
@@ -3,38 +3,50 @@
     {
       "id": 289,
       "title": "I need help with this 1",
+      "description": "Need a hand with tyres",
+      "requester": "me@mail.com",
```

Comment:
Me and Erik have been working on the serializers, we will make sure they match this.